### DISCUSSION TITLE
Git Gui A link to the official website is required.

### DISCUSSION THREAD

**Discussion by jarupxx** [git-for-windows/git#4889] [Q&A] [answered]
- [x] I was not able to find an [open](https://github.com/git-for-windows/git/issues?q=is%3Aopen) or [closed](https://github.com/git-for-windows/git/issues?q=is%3Aclosed) issue matching what I'm seeing

### Setup

 - Which version of Git for Windows are you using? Is it 32-bit or 64-bit?

```
$ git --version --build-options

git version 2.44.0.windows.1
cpu: x86_64
built from commit: ad0bbfffa543db6979717be96df630d3e5741331
sizeof-long: 4
sizeof-size_t: 8
shell-path: /bin/sh
feature: fsmonitor--daemon
```

 - Which version of Windows are you running? Vista, 7, 8, 10? Is it 32-bit or 64-bit?

```
$ cmd.exe /c ver

Microsoft Windows [Version 10.0.22631.3296]
```

### Details

It would be nice if Git Gui had a link to the official website to find new versions. We can usually access it from a menu ```Help``` or ```About```.

**Comment by enderwert** [comment]
****

**Comment by dscho** [comment, marked as answer]
This is not a Windows-specific issue. You may have more luck taking it to [the Git mailing list](mailto:git@vger.kernel.org) (send plain-text messages, HTML messages are dropped silently).

Having said that: the tl;dr is that there is none.

Git GUI is looking for a new maintainer, the current maintainer hosts the project at https://github.com/prati0100/git-gui/ but does not release anything there, instead letting the Git project handle that (which, as mentioned above, is run via a mailing list).

**Reply by jarupxx** [reply]
Oh, I see. I will try my best to contribute to Git GUI.
